modules/agar/src/main/ts/ephox/agar/api/Generators.ts

```diff
@@ -8,18 +8,18 @@ import * as TagDecorator from '../arbitrary/TagDecorator';
 
 const selection = (container, exclusions) => GenSelection.selection(container, exclusions);
 
-const describeSelection = (root, generated) =>
-  Hierarchy.path(root, generated.start()).bind((startPath) =>
-    Hierarchy.path(root, generated.finish()).map((finishPath) => ({
+const describeSelection = (root: SugarElement<HTMLElement>, generated: SimRange) =>
```

Comment:
These changes don't look related to anything here? This is also been fixed already and with other larger changes in https://github.com/tinymce/tinymce/pull/7438 so I'd suggest reverting this.

Reply:
It was causing issues when I changed the parser over. Seeing as it's already in another branch though, I've reverted it.